### PR TITLE
modal dialog alignment, left without image, centered with image

### PR DESCRIPTION
**Pull request by faustoonna**
changing image, title and description alignment:
- has image, center aligned
-  no image, left aligned

### CODE REVIEW

**Review by raul-onna — APPROVED** [review]

File: projects/pastanaga-angular/src/lib/modal/modal-dialog/modal-dialog.component.scss [hunk 1]
```diff
@@ -9,7 +9,11 @@ $header-height-medium: $padding-modal + $dimension-image + $spacing-dialog-secti
     .pa-modal-header {
         @include fixed-header;
         padding: $padding-modal $padding-modal $spacing-dialog-sections;
-        text-align: center;
+        text-align: left;
+
+        &-centered {
```
Comment (mpellerin42) [inline projects/pastanaga-angular/src/lib/modal/modal-dialog/modal-dialog.component.scss:14]: I would recommend not to use this kind of shortcut: it makes it hard to find CSS classes using the IDE navigation and search tools.
Better to write the whole class name

File: projects/pastanaga-angular/src/lib/modal/modal-dialog/modal-dialog.component.scss [hunk 2]
```diff
@@ -19,6 +23,10 @@ $header-height-medium: $padding-modal + $dimension-image + $spacing-dialog-secti
 
         .pa-modal-description {
             margin-top: $spacing-dialog-sections;
+            text-align: left;
+            &-centered {
```
Comment (mpellerin42) [inline projects/pastanaga-angular/src/lib/modal/modal-dialog/modal-dialog.component.scss:27]: same here

**Review by mpellerin42 — CHANGES_REQUESTED** [review]

**Review by mpellerin42 — APPROVED** [review]

File: projects/pastanaga-angular/src/lib/modal/modal-dialog/modal-dialog.component.scss (hunk 1 above)
Comment (faustoonna) [reply projects/pastanaga-angular/src/lib/modal/modal-dialog/modal-dialog.component.scss]: done

File: projects/pastanaga-angular/src/lib/modal/modal-dialog/modal-dialog.component.scss (hunk 2 above)
Comment (faustoonna) [reply projects/pastanaga-angular/src/lib/modal/modal-dialog/modal-dialog.component.scss]: done

**Pull request merged**